app/views/broadcast_announcements/_form.html.erb

```diff
@@ -0,0 +1,41 @@
+  <%= simple_form_for @broadcast_announcement, html: { class: 'form-horizontal' } do |f| %>
+    <%= f.error_notification %>
+    <section class="content">
+      <div class="container-fluid">
+        <div class="row">
+          <!-- left column -->
+          <div class="col-md-12">
+            <!-- jquery validation -->
+            <div class="card card-primary">
+              <!-- /.card-header -->
+              <!-- form start -->
+              <div class="card-body w-50">
+                <%= f.hidden_field :user_id, value: current_user.id %>
+                <%= f.hidden_field :organization_id, value: current_organization.id %>
+                <%= f.label :message, "Announcement " %><i>(500 character max)</i>
+                <%= f.input :message, label: false, maxlength: 500, autofocus: true %>
+                <%= f.label :link, "URL for more information" %>
+                <%= f.text_field :link, label: false, class: "form-control" %>
+                <br /><br />
+                <input id="expiry_check" name="expiry_check" type="checkbox" 
+                      onclick="var select_input = Array.from(document.getElementsByTagName('select'));
+                        if(this.checked) {
+                        select_input.forEach((input) => {input.disabled = false});
+                        } else {
+                        select_input.forEach((input) => {input.disabled=true});
+                        }" >
+                <label for="expiry_check">Expiration date</label>
+                <%= f.input :expiry, label: false, disabled: true, class: "form-control" %>
```

Comment:
You should be using a calendar control here - then you don't need the checkbox. You can use Litepicker for this: https://github.com/rubyforgood/human-essentials/blob/2c7bd3d70f0e7f46a31e9d6e1872f2fa195de1ca/app/javascript/application.js#L88